api/v2/schema.py

```diff
@@ -13,7 +18,7 @@ class V2ScoreResponse(Schema):
     expiration_timestamp: Optional[str]
     threshold: Decimal
     error: Optional[str]
-    stamp_scores: Optional[Dict[str, Decimal]]
+    stamps: Optional[Dict[str, Dict]]
```

Comment:
I would define this as `tamps: Optional[Dict[str, V2ScoreResponseStampInfo]]` and define the `V2ScoreResponseStampInfo` model